### ISSUE TITLE
Install Issue2

### ISSUE THREAD

**Issue by skellyton5** [PokemonGoF/PokemonGo-Bot#1313]
I installed PyYaml

What now...

I'm using an older version, where I open a command window and type in a bunch of settings, but when I try to do the same with the new version it doesn't work.

**Comment by vicch** [comment]
It would be helpful if you post the commands you  executed and the outputs here, or do a screenshot.

**Comment by skellyton5** [comment]
http://i.imgur.com/XuVmfMy.png

**Comment by vicch** [comment]
Run this and try again.
```
pip install -r requirements.txt
```

**Comment by skellyton5** [comment]
I already did this for the old version (Which is currently running fine, but still glitchy)

But when I do it again, for the new version's requirements.txt I get this error.

http://i.imgur.com/kdZr3rr.png

When Looking into the actual requirements.txt file, It looks like this

http://imgur.com/a/Frw7h

The top one is wierd, not like the others, I'ts the one causing issues.
I deleted it, ran it again, Had most things already installed except for colorama. 
Tried to install the top item with pip, it failed. 
Still have the same issue.
